isomorphe/doc/index.md

```diff
@@ -1,13 +1,84 @@
-# ecospheres-isomorphe
+## Table des matières
 
-## Pourquoi ISOMorphe ?
+- [Pourquoi ISOmorphe ?](#pourquoi)
+- [Comment fonctionne ISOmorphe ?](#comment)
+- [Comment utiliser ISOmorphe ?](#tutoriel)
+- [Transformations disponibles](#transformations)
```

Comment:
Should we move this in first position?

Why?
- It's a small section buried at the bottom. Might easily be overlooked.
- https://github.com/ecolabdata/ecospheres-isomorphe/pull/84/commits/ade5f41e8f20b61960bdefc36150f7d0ba38ab85 links to it but since it's a small section at the bottom, the top of the page doesn't align with the section header.
- Once users are familiar with the tool, that's what they'll most be interested in.
- It's a small section, so it doesn't matter so much if it's at the top.

Reply:
I'm fine either way. Feels like a technical / reference section that belongs naturally at the bottom, but it also might be the most often looked up section.

Reply:
Moved it at the top, it's too easy to miss at the bottom.